k8s/sandbox/common/idam-sprod/idam-web-public.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: idam-web-public
+  namespace: idam-sprod
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:preview-*
+spec:
+  releaseName: idam-web-public
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: idam-web-public
+    version: 0.2.3
+  values:
+    java:
+      image: hmctspublic.azurecr.io/idam/web-public:preview
+      ingressHost: idam-web-public-aks.sprod.platform.hmcts.net
+      aadIdentityName: idam-sprod
+      replicas: 1
```

Comment:
You'll need to add environment variable overrides here. Like https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/common/idam/idam-web-public.yaml#L24-L25

Reply:
done